libs/brig-types/src/Brig/Types/Team/LegalHold.hs

```diff
@@ -84,32 +84,62 @@ instance FromJSON LegalHoldService where
                    <*> o .: "fingerprint"
                    <*> o .: "auth_token"
 
-data ViewLegalHoldService = ViewLegalHoldService
-    { viewLegalHoldServiceTeam        :: !TeamId
-    , viewLegalHoldServiceUrl         :: !HttpsUrl
-    , viewLegalHoldServiceFingerprint :: !(Fingerprint Rsa)
-    }
+data ViewLegalHoldService
+    = ViewLegalHoldService ViewLegalHoldServiceInfo
+    | ViewLegalHoldServiceNotConfigured
+    | ViewLegalHoldServiceDisabled
   deriving stock (Eq, Show, Generic)
 
 instance ToJSON ViewLegalHoldService where
-    toJSON s = object
-        $ "team_id"     .= viewLegalHoldServiceTeam s
-        # "base_url"    .= viewLegalHoldServiceUrl s
-        # "fingerprint" .= viewLegalHoldServiceFingerprint s
-        # []
+    toJSON s = case s of
+        ViewLegalHoldService info -> object
+            $ "status" .= String "configured"
+            # "info"   .= toJSON info
```

Comment:
Pretty sure you don't need the `toJSON` 😄 
Don't need to bother removing it though